CA_french/astrophysics_fr.properties

```diff
@@ -70,7 +70,7 @@ datasetfieldtype.coverage.Temporal.StopTime.watermark=AAAA-MM-JJ
 datasetfieldtype.coverage.Spatial.watermark=
 datasetfieldtype.coverage.Depth.watermark=Entrer un nombre � virgule flottante.
 datasetfieldtype.coverage.ObjectDensity.watermark=Entrer un nombre � virgule flottante.
-datasetfieldtype.coverage.ObjectCount.watermark=Entrer un nombre � virgule flottante.
+datasetfieldtype.coverage.ObjectCount.watermark=Entrer un nombre entier.
```

Comment:
Yes indeed, English file says "Enter an integer". Thanks for spotting this error.